src/components/navigation/NavigationPanel.scss

```diff
@@ -12,7 +12,7 @@
     margin-bottom: 2rem;
 
     .motor-logo {
-      width: 100px;
+      height: 60px;
     }
   }
 
```

Comment:
Please Kindly remove the Nav panel's border apply active class to each nav link item and remove hover. Please apply certain box shadow - minimum 
![890C3EEA-5E58-494A-AB1F-06DD2F21625C](https://github.com/binodbhusal/final_capstone_frontend/assets/124040422/e1430bc7-5b43-453e-80a4-6a6ab324c923)